### PR TITLE
feat(usermanual): Add out-of-office/absence message feature

### PR DESCRIPTION
**Pull request by ChristophWurst**
### ☑️ Resolves

* Fix https://github.com/nextcloud/documentation/issues/11333
* Close https://github.com/nextcloud/server/issues/40397

### 🖼️ Screenshots

Bildschirmfoto vom 2023-12-20 13-16-23
Bildschirmfoto vom 2023-12-20 13-31-08

### CODE REVIEW

File: user_manual/groupware/absence.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Setting out-of-office messages
+==============================
+
+If you are absent for vacation, sick leave or similar, you can add an out-of-office message in the **Absence** section of the **Availability** settings.
+
+The interface asks for the time of absence, a short and a long message. This data is used for the following purposes:
+
+1) Your user status will change to the short message when your absence starts and reset when it ends.
+2) And event with status *busy* will be created in your personal calendar. This allows others to see that you are not available when they use the free/busy feature.
+3) If enabled, the Mail app will apply an autoresponder.
```
Comment (miaulalala) [inline user_manual/groupware/absence.rst:13]: Can you elaborate that the short message will be the subject and the long message the mail body here? (IIRC from the email I have received)

File: user_manual/groupware/absence.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+4) The Talk app will show the out-of-office message to others when they try to reach you in a 1:1 chat during your absence.
```
Comment (miaulalala) [inline user_manual/groupware/absence.rst:14]: Same here, the long message will be shown as the text IIRC

**Review by miaulalala — DISMISSED** [review]
Some comments

File: user_manual/groupware/absence.rst (hunk 1 above)
Comment (ChristophWurst) [reply user_manual/groupware/absence.rst:13]: That is something I wanted to clarify, hence the vague documentation. I'm not sure if short message should even be used as subject.

Comment (ChristophWurst) [reply user_manual/groupware/absence.rst:13]: https://github.com/nextcloud/mail/issues/8925#issuecomment-1864599786 let's see ;)

Comment (ChristophWurst) [reply user_manual/groupware/absence.rst:13]: https://github.com/nextcloud/mail/issues/9219

Comment (st3iny) [reply user_manual/groupware/absence.rst:13]: This is still a bit vague. We only use the long message as the body after https://github.com/nextcloud/mail/pull/9228.

File: user_manual/groupware/mail.rst [hunk 3]
```diff
@@ -237,3 +239,16 @@ Snoozing a message or thread moves it into a dedicated mailbox until the selecte
...
+---------------------------
+
+The Mail app has a simple editor for Sieve scripts and an interface to configure autoresponders. Sieve has to be enabled in the :ref:`account settings <mail-account-settings>`.
+
+Autoresponders
+~~~~~~~~~~~~~~
+
+.. versionadded:: 3.5 Autoresponder can follow system settings.
+
+The autoresponder is off by default. It can be set manually, or follow the system settings. Following system settings means that information entered on the :ref:`Absence settings section <groupware-absence>` is applied automatically.
```
Comment (st3iny) [inline user_manual/groupware/mail.rst:254]: Only the long message is used and the subject is built from the old one. That should be mentioned here.

Comment (st3iny) [reply user_manual/groupware/mail.rst:254]: Ref https://github.com/nextcloud/mail/pull/9228

**Review by GretaD — APPROVED** [review]

**Pull request merged**